test/extensions/CompoundTree.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+	<head>
+		<meta charset="utf-8">
+		<title>Test Grid with Compund Columns, Column Sets and Tree</title>
+		<meta name="viewport" content="width=570">
+		<style>
+			@import "../../../dojo/resources/dojo.css";
+			@import "../../css/dgrid.css";
+
+			.dgrid {
+				width: 750px;
+				margin: 10px;
+			}
+
+			.dgrid .dgrid-content .dgrid-cell {
+				height: 24px;
+			}
+		</style>
+		<script src="../../../dojo/dojo.js" data-dojo-config="async: true"></script>
+	</head>
+	<body >
+		<div id="grid"></div>
+
+		<script>
+			require(["dgrid/Tree", "dgrid/ColumnSet", "dgrid/OnDemandGrid",
+						"dgrid/extensions/CompoundColumns", "dstore/Memory"],
```

Comment:
I've replaced `Memory` and the data on this page with `createHierarchicalStore` and `hierarchicalCountryData` like what's used in `Tree.html`.  Incidentally that's also what made the `level`-always-zero issue more apparent to me, since it's more obvious with nested parents.